src/bin/serve.rs

```diff
@@ -0,0 +1,13 @@
+use std::error::Error;
+
+#[path="../main.rs"] mod main;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    main::main()?;
+
+    let footer = format!("{} {}", env!("CARGO_PKG_NAME"), env!("CARGO_PKG_VERSION"));
+
+    warpy::server::run(format!("{}/site", env!("CARGO_MANIFEST_DIR")), [0, 0, 0, 0], footer, Some(8000), false).await?;
```

Comment:
would it be worth allowing an env override of the port? i know i often end up forgetting things running on machine, particularly docker containers, and 8000 is obviously fairly common. think it'd be nice to be able to change this instead of having to first go kill something already on that port